pkg/gadgets/seccomp/gadget.go

```diff
@@ -50,6 +56,14 @@ type TraceSingleton struct {
 
 var traceSingleton TraceSingleton
 
+func (f *TraceFactory) SupportsOutputMode(outputMode string) bool {
+	return outputMode == "Status" || outputMode == "ExternalResource"
+}
+
+func (f *TraceFactory) AddToScheme(scheme *apimachineryruntime.Scheme) {
+	utilruntime.Must(seccompprofilev1alpha1.AddToScheme(scheme))
```

Comment:
Could you please explain me why it's needed? Is it because we're going to use the client to modify that kind of resource?

Reply:
Exactly. Otherwise, the client does not manage to read/write that kind of resource. Alternatively, the gadget could create its own client but I thought it is simpler this way.